projects/lib/styles/ui-builder.scss

```diff
@@ -15,10 +15,29 @@ uib-toolbar {
   z-index: 1046; // just above the configurator
 }
 
-/* container is a flexbox */
-.uib-container {
-  display: flex;
+/* zone's styles */
+@import "_zone";
+
+.dropzone {
+  flex-grow: 1;
+
+  min-width: 32px;
+  min-height: 32px;
 }
 
-/* zone's styles */
-@import "_zone";
\ No newline at end of file
+.dropzone-content {
```

Comment:
dropzone and dropzone-content should have uib- prefix to make it clear they are part of the ui-builder library